paper/paper.Rmd

```diff
@@ -37,7 +37,7 @@ affiliations:
   name: Monash University
 ---
 
-```{r}
+```{r Document Setup}
 #| label: setup
 #| echo: false
 #| message: false
```

Comment:
This chunk already has the `label` paramenter on line 41, so you don't need to have "Document Setup" in the chunk header. I'm also fairly sure that you can't have spaced in label names.

Reply:
Oops. I'll revert the changes. Thank you!

Reply:
Unfortunately removing the label in the chunk header means the code chunks don't show their labels in VS Code's outline viewer (it shows up as "unnamed-chunk" instead). The labels only shows up when I include them in the chunk headers.